ci/ReleaseTool/PrepCommand.cs

```diff
@@ -155,8 +154,7 @@ public int Run()
                     if (!gitClient.LocalBranchExists($"origin/{options.ReleaseBranch}"))
                     {
                         gitClient.Fetch();
-                        gitClient.CheckoutRemoteBranch(options.CandidateBranch);
-                        gitClient.Commit(string.Format(ReleaseBranchCreationCommitMessageTemplate, options.Version));
```

Comment:
How come we removed this commit?

Reply:
So, this is actually a fix:
If creating a release branch, if we created it from the Candidate branch and then added a commit, the resultant release branch would actually be *ahead* of the candidate branch and the PR cannot be created (rejected by GitHub).
I just opted for not adding any commits and creating from the source branch instead.